kategory-test/src/main/kotlin/kategory/laws/FunctorFilterLaws.kt

```diff
@@ -0,0 +1,31 @@
+package kategory.laws
+
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+import kategory.*
+
+object FunctorFilterLaws {
+
+    inline fun <reified F> laws(AP: Applicative<F> = applicative(), FFF: FunctorFilter<F> = functorFilter(), EQ: Eq<HK<F, Int>>): List<Law> =
```

Comment:
FunctorLaws has 2 constructor, one convenient one with Applicative in the case that you're testing anything that inherits from applicative, and another that takes a constructor function. In this case as we're testing something that doesn't inherit from applicative please use the second one.